runtime/doc/ui.txt

```diff
@@ -52,6 +52,11 @@ with these (optional) keys:
 	`term_name`		Sets the name of the terminal 'term'.
 	`term_colors`		Sets the number of supported colors 't_Co'.
 	`term_background`	Sets the default value of 'background'.
+	`stdin_fd`		Read buffer from `fd` as if it was a stdin pipe
```

Comment:
would `ext_stdin` make sense as the name? Seems like it would, since it's similar to the other "ext_" options in that it is a UI shim.